### DISCUSSION TITLE
some classes not working

### DISCUSSION THREAD

**Discussion by misheck12** [tailwindlabs/tailwindcss#9752] [Help]
Hello guys I have tried to use tailwind in react. unfortunately, some classes are not working is there any way I can solve this?

**Comment by wongjn** [comment]
Do you have a remotely-hosted git repository (or similar) you can share? Which classes do and don't work for you? It's difficult to identify the problem without any details!

You can also try your DOM in [Tailwind Play](https://play.tailwindcss.com) – if it works in there then it could indicate some problem with your project configuration.

**Reply by misheck12** [reply]
Hello @wongjn here is my repo you can check and confirm with me if I did it in the right way or not https://github.com/misheck12/amenshi_4_life/tree/Navbar

**Reply by wongjn** [reply]
Which classes don't work?

**Comment by diinhoalmeida** [comment]
[misheck12](https://github.com/misheck12), i checked your project and find the posible solution!

1- Create a folder style into src;
2- Create a file main.css into style folder;
3- Put the code below inside the main.css file:

````
@tailwind base;
@tailwind components;
@tailwind utilities;
```` 

4- Import main.css into App.js file:
````
import './styles/main.css';
````
